src/Symfony/Component/Security/Tests/Acl/Voter/AclVoterTest.php

```diff
@@ -27,7 +27,7 @@ class AclVoterTest extends \PHPUnit_Framework_TestCase
      */
     public function testSupportsAttribute($attribute, $supported)
     {
-        list($voter, , $permissionMap, ,) = $this->getVoter(true, false);
+        list($voter, , $permissionMap, , ) = $this->getVoter(true, false);
```

Comment:
This change looks wrong to me. Actually, do we need those extra `,` at the end of the list (I think we can omit them).